File: src/pages/exercises/function-fibonacci-number/_codes/ruby/code/fibonacci-number.spec.rb
require_relative 'fibonacci-number.rb'

describe 'Fibonacci series' do
  context 'making the 0th number' do
    it nil do
      expect(fibonacci(0)).to eql(nil)
    end
  end 

  context 'making the 1st numbers' do
    it 0 do
      expect(fibonacci(1)).to eql(0)
    end
  end 

  context 'making the 2nd numbers' do
    it 1 do
      expect(fibonacci(2)).to eql(1)
    end
  end 

  context 'making the 4th numbers' do
    it 2 do
      expect(fibonacci(4)).to eql(2)
    end
  end 

  context 'making the 6th numbers' do
    it 5 do
      expect(fibonacci(6)).to eql(5)
    end
  end 
end 

File: src/pages/exercises/function-fibonacci-number/_codes/php/code/spec/fibonacci-number.spec.php
<?php

require __DIR__ . '/../src/fibonacci-number.php';

describe('Fibonacci series', function () {
  it('making the 0th number', function () {
    expect(fibonacci(0))->toBeNull();
  });

  xit('making the 1st numbers', function () {
    expect(fibonacci(1))->toBe(0);
  });

  xit('making the 2nd numbers', function () {
    expect(fibonacci(2))->toBe(1);
  });

  xit('making the 4th numbers', function () {
    expect(fibonacci(4))->toBe(2);
  });

  xit('making the 6th numbers', function () {
    expect(fibonacci(6))->toBe(5);
  });
});
